### PR TITLE
FEATURE: TMA-1034 Create image build pipeline

### PR DESCRIPTION
**Pull request by sadam21**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/671/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1498/ : FAILURE in 2m 07s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1759/ : FAILURE in 3m 00s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/672/ : SUCCESS in 17s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1499/ : FAILURE in 1m 14s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1760/ : FAILURE in 1m 35s

**Comment by michalvanco** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/673/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1500/ : SUCCESS in 2m 26s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1761/ : FAILURE in 1m 08s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-check-pipeline/1/ : FAILURE in 18s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (michalvanco) [inline Jenkinsfile]: remove line

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/676/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1503/ : SUCCESS in 1m 12s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1764/ : FAILURE in 1m 13s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-check-pipeline/4/ : SUCCESS in 6m 08s

File: Jenkinsfile (hunk 1 above)
Comment (sadam21) [reply Jenkinsfile]: Copyrights which I removed by mistake were added back :-)

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,5 @@
 FROM harbor.intgdc.com/tools/gdc-java-8-jre:b057b53
 
-MAINTAINER LCM <lcm@gooddata.com>
```
Comment (Hahihula) [inline Dockerfile]: Why are you removing the maintainer ?

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/679/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1506/ : FAILURE in 1m 06s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1767/ : SUCCESS in 1m 05s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-check-pipeline/6/ : SUCCESS in 5m 25s

File: Dockerfile (hunk 2 above)
Comment (sadam21) [reply Dockerfile]: Because it's deprecated https://docs.docker.com/engine/reference/builder/#maintainer-deprecated .

Comment (Hahihula) [reply Dockerfile]: ok.. good to know

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/680/ : SUCCESS in 50s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1507/ : FAILURE in 1m 14s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1768/ : SUCCESS in 1m 11s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-check-pipeline/7/ : SUCCESS in 6m 10s

**Pull request merged**